app/views/find/v2/results/index.html.erb

```diff
@@ -16,7 +16,7 @@
   <%= form.hidden_field :utm_source, value: "results" %>
 
   <div class="app-filter-layout">
-    <div class="app-filter-layout__filter app-filter">
+    <div class="app-filter-layout__filter app-filter" hidden="hidden">
```

Comment:
Tested and on mobile everything worked.

Although for non JS users the filter disappears because of this hidden attribute. Is there a way to add this attributes on the JS instead of hardcoding on document.load?

Reply:
Hmm, true...

It really just needs to be hidden for mobile. I might need to mess with the CSS.